dace/sdfg/state.py

```diff
@@ -421,7 +421,8 @@ def memlet_path(self, edge: MultiConnectorEdge[mm.Memlet]) -> List[MultiConnecto
             # Trace through scope entry using IN_# -> OUT_#
             if isinstance(curedge.dst, (nd.EntryNode, nd.ExitNode)):
                 if curedge.dst_conn is None:
-                    raise ValueError("Destination connector cannot be None for {}".format(curedge.dst))
+                    #raise ValueError("Destination connector cannot be None for {}".format(curedge.dst))
```

Comment:
Is there a reason for removing this check?

Reply:
I probably forgot it when I first trying to implement where size arrays and arrays had different containers. Must have forgotten it like that.

I reran some tests locally, does not seem to cause an issue.